.github/workflows/recipe-test.yaml

```diff
@@ -0,0 +1,34 @@
+name: Recipe Test
```

Comment:
This is good for now, but FYI once everything is working we will **replace** the test that uses `drush si minimal --existing-config -y`